### PR TITLE
add tile count label

### PR DESCRIPTION
**Pull request by uniemimu**
Signed-off-by: Ukri Niemimuukko <ukri.niemimuukko@intel.com>

### CODE REVIEW

File: cmd/gpu_nfdhook/labeler.go [hunk 1]
```diff
@@ -114,17 +114,18 @@ func fallback() uint64 {
 }
 
 // getMemoryAmount reads the GPU memory amount from the system.
-func (l *labeler) getMemoryAmount(gpuName string) uint64 {
+func (l *labeler) getMemoryAmount(gpuName string) (uint64, uint64) {
```
Comment (rojkov) [inline cmd/gpu_nfdhook/labeler.go:117]: Now the function name doesn't fully reflect what it actually returns. I guess the return values should be documented somehow for better readability. Either in the doc string or by giving them names. 

The rest looks good!

**Review by bart0sh — DISMISSED** [review]
/lgtm

**Comment by bart0sh** [comment]
@uniemimu unit tests seem to be failing. can you have a look?

**Comment by uniemimu** [comment]
> @uniemimu unit tests seem to be failing. can you have a look?

Oops. Fixed.

**Review by mythi — APPROVED** [review]

**Review by bart0sh — APPROVED** [review]

**Pull request merged**